src/js/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -105,27 +80,41 @@ class YourClaimsPageV2 extends React.Component {
   }
 
   renderErrorMessages() {
-    const { claimsLoading, appealsLoading, appealsAvailable, canAccessAppeals, canAccessClaims, claimsAvailable, claimsAuthorized } = this.props;
+    const {
+      claimsLoading,
+      appealsLoading,
+      appealsAvailable,
+      canAccessAppeals,
+      canAccessClaims,
+      claimsAvailable,
+      // claimsAuthorized
+    } = this.props;
 
-    if (claimsLoading && appealsLoading) {
+    if (claimsLoading || appealsLoading) {
       return null;
     }
 
     if (canAccessAppeals && canAccessClaims) {
-      if (!claimsAvailable && !appealsAvailable) {
+      if (claimsAvailable !== claimsAvailability.AVAILABLE
+          && appealsAvailable !== appealsAvailability.AVAILABLE) {
         return <ClaimsAppealsUnavailable/>;
       }
     }
 
-    if (canAccessClaims) {
-      if (!claimsAvailable) {
-        return <ClaimsUnavailable/>;
-      } else if (!claimsAuthorized) {
-        return <ClaimsUnauthorized/>;
-      }
+    // TO-DO: Figure out if this is still something we want
+    // if (canAccessClaims) {
+    //   if (claimsAvailable !== claimsAvailability.AVAILABLE) {
+    //     return <ClaimsUnavailable/>;
+    //   } else if (!claimsAuthorized) {
+    //     return <ClaimsUnauthorized/>;
+    //   }
+    // }
+
+    if (canAccessClaims && claimsAvailable !== claimsAvailability.AVAILABLE) {
```

Comment:
We talked about this. The conclusion we came to is to check if the claims (or appeals) are still loading before rendering the error message.